### PR TITLE
contrib/intel/jenkins: Add ze ci testing [for testing only DO NOT MERGE]

### PR DESCRIPTION
**Pull request by zachdworkin**
Add ze support to jenkins python scripts

Signed-off-by: zdworkin <zachary.dworkin@intel.com>

### CODE REVIEW

File: contrib/intel/jenkins/Jenkinsfile [hunk 1]
```diff
@@ -122,6 +122,21 @@ pipeline {
...
+                          sh """
+                            env
+                            (
+                                cd  ${env.WORKSPACE}/contrib/intel/jenkins/
+                                python3.7 runtests.py --prov=shm --ofi_build_mode='dbg'
+                            )
+                          """
+                        }
+		     }
+		 }
```
Comment (shefty) [inline contrib/intel/jenkins/Jenkinsfile:139]: CI testing only, I know, but could you align this block with the other blocks are the same level

File: contrib/intel/jenkins/run.py [hunk 2]
```diff
@@ -60,6 +60,17 @@ def shmemtest(core, hosts, mode, util=None):
               .format(runshmemtest.testname))
     print("----------------------------------------------------------------------------------------\n")
 
+def zetest(core, hosts, mode, util=None):
+    runzetest = tests.ZeTest(jobname=jbname,buildno=bno,\
```
Comment (shefty) [inline contrib/intel/jenkins/run.py:64]: "runzetest"  -- this is why I love the 'ze' prefix. :)

File: contrib/intel/jenkins/Jenkinsfile (hunk 1 above)
Comment (zachdworkin) [reply contrib/intel/jenkins/Jenkinsfile:139]: yes

File: contrib/intel/jenkins/Jenkinsfile [hunk 3]
```diff
@@ -122,6 +122,23 @@ pipeline {
...
+                 stage('ze-shm') {
+                     agent {node {label 'eth'}}
+                     steps{
+                         withEnv(['PATH+EXTRA=/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/bin/:$PYTHONPATH'])
+                         {
+                           sh """
+                             env
+                             (
+                                 cd ${env.WORKSPACE}/contrib/intel/jenkins/
+                                 python3.7 runtests.py --prov=shm --ofi_build_mode='dbg'
```
Comment (nikhilnanal) [inline contrib/intel/jenkins/Jenkinsfile:135]: Once  it is verified that this test is working, we can remove the similar call from the eth-udp-shm stage.

File: contrib/intel/jenkins/Jenkinsfile [hunk 4]
```diff
@@ -122,6 +122,23 @@ pipeline {
...
+                             )
+                           """
+                         }
+                      }
+                 }
+
                  stage('hfi1-psm2-verbs-dbg') {
```
Comment (shefty) [inline contrib/intel/jenkins/Jenkinsfile:142]: Maybe it's time to remove the hfi1 and psm2 testing from our per PR CI.  We can expand what we do test on the remaining providers.

**Comment by shefty** [comment]
The PR indicates this is for CI testing only, with the last update over 3 weeks ago.  Assuming this PR is not intended for merging or active anymore.

**Pull request closed without merging**